lib/fluent/output.rb

```diff
@@ -450,9 +450,17 @@ def initialize
       super
     end
 
+    def format_stream(tag, es)
+      if @time_as_integer
```

Comment:
`time_as_integer` doesn't have this option so this commit should not be backported to v0.12